git_machete/constants.py

```diff
@@ -0,0 +1,32 @@
+from typing import Tuple
+
+import os
+
+MAX_COUNT_FOR_INITIAL_LOG = 10
+DISCOVER_DEFAULT_FRESH_BRANCH_COUNT = 10
+
+PICK_FIRST_ROOT: int = 0
+PICK_LAST_ROOT: int = -1
+
+BRANCH_DEF = Tuple[str, str]
```

Comment:
Hmm type alias doesn't feel right here among there numeric/string constants, I'd rather put it back to cmd.py